### PR TITLE
Added extra plugin information node to the order creation

### PR DESCRIPTION
**Pull request by dmitrijev**

### CODE REVIEW

File: src/Order.php [hunk 1]
```diff
@@ -86,6 +88,11 @@
     private $customer;
 
     /**
+     * @var Extra|null
```
Comment (ghost) [inline src/Order.php:93]: Would be nice to have a description of the purpose of the 'extra' field.

**Review by ghost — COMMENTED** [review]
What's the purpose of creating a class for plugin info in the 'extra' field? The idea behind the 'extra' field is that it holds arbitrary unstructured data, as defined by the client. Should the SDK really define structure here?

**Comment by ghost** [comment]
Can you give an example about how to use the new extra field?

**Comment by dmitrijev** [comment]
You mean readme.txt example?

**Comment by ghost** [comment]
Just a comment here should be enough, I'm curious what it would look like.

**Comment by dmitrijev** [comment]
``` PHP
$order = $client->createIdealOrder(
    2500,                           // The amount in cents
    'EUR',                          // The currency
    'INGBNL2A',                     // The iDEAL issuer
    'A great order',                // A description (optional)
    'order-234192',                 // Your identifier for the order (optional)
    'http://www.example.com',       // The return URL (optional)
    'PT15M',                         // The expiration period in ISO 8601 format (optional)
    null,                             // Customer information
   ['extra_variable' => 'Hello World'] // Extra Order information 
);
```

**Review by ghost — APPROVED** [review]

**Pull request merged**